protocols/kad/src/record/store.rs

```diff
@@ -30,14 +31,15 @@ use std::borrow::Cow;
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// The possible errors of a `RecordStore` operation.
-#[derive(Debug)]
+#[derive(Error, Debug, Clone)]
 pub enum Error {
-    /// The store is at capacity w.r.t. the total number of stored records.
+    #[error("the store cannot contain any more records")]
```

Comment:
I would be in favor of also keeping the code docs. `#[error(..)]` only sets the `Display` message for this type, but isn't visible to a user that is looking at this type through the API docs on docs.rs.

Reply:
done